website/docs/topics/non-openai-models/about-using-nonopenai-models.md

```diff
@@ -0,0 +1,53 @@
+# Non-OpenAI Models
+
+AutoGen allows you to use non-OpenAI models through proxy servers that provide
+an OpenAI-compatible API or a [custom model client](https://microsoft.github.io/autogen/blog/2024/01/26/Custom-Models)
+class.
+
+Benefits of this flexibility include access to hundreds of models, assigning specialized
+models to agents (e.g., fine-tuned coding models), the ability to run AutoGen entirely
+within your environment, utilising both OpenAI and non-OpenAI models in one system, and cost
+reductions in inference.
+
+## OpenAI-compatible API proxy server
+Any proxy server that provides an API that is compatible with [OpenAI's API](https://platform.openai.com/docs/api-reference)
+will work with AutoGen.
+
+These proxy servers can be cloud-based or running locally within your environment.
+
+![Cloud or Local Proxy Servers](images/cloudlocalproxy.png)
+
+### Cloud-based proxy servers
+By using cloud-based proxy servers, you are able to use models without requiring the hardware
+and software to run them.
+
+These providers can host open source/weight models, like [Hugging Face](https://huggingface.co/),
+or their own closed models.
+
+See [Cloud-based Proxy Servers](cloud-based-proxy-servers) for a guide on how to
```

Comment:
We can list the pages on cloud based servers here. People can add more in the future.

Reply:
Good suggestion, I've put the cloud and local proxy server examples into their own pages and referenced it from this page.